### PR TITLE
Contact tests from nucypher repo

### PR DESCRIPTION
**Pull request by vzotova**
Moves tests from nucypher repo + adjusting them for using brownie
Opens #26

### CODE REVIEW

**Review by theref — APPROVED** [review]

**Review by manumonti — CHANGES_REQUESTED** [review]
Wow! Nice work!

**Comment by KPrasch** [comment]
Awesome!

**Review by derekpierre — APPROVED** [review]
🎸

File: tests/application/test_operator.py [hunk 1]
```diff
@@ -0,0 +1,344 @@
...
+
+CONFIRMATION_SLOT = 1
+MIN_AUTHORIZATION = Wei("40_000 ether")
+MIN_OPERATOR_SECONDS = 24 * 60 * 60
+NULL_ADDRESS = brownie.convert.to_address("0x" + "0" * 40)  # TODO move to some test constants
+
+
+def test_bond_operator(accounts, threshold_staking, pre_application, chain):
+    accounts.add()
+    creator, staking_provider_1, staking_provider_2, staking_provider_3, staking_provider_4, operator1, operator2, \
```
[the remark this answers is not in the document]
Comment (vzotova) [reply tests/application/test_operator.py]: There is small issue with pre-commit hooks with this file, after prettifying it moves all variables at one line and fails when check line length. I commited with 'no-verify' and this is why missed error.
Anyway I changed line
✔️

[the remark this answers is not in the document]
Comment (manumonti) [reply tests/application/test_operator.py]: Super!

**Review by manumonti — APPROVED** [review]

**Review by cygnusv — APPROVED** [review]

**Pull request merged**